### ISSUE TITLE
0.49.7 Uncaught java.lang.NullPointerException mid way through a match

### ISSUE THREAD

**Issue by Bows3r** [MegaMek/megamek#3653]
### Environment
0.49.7. Standalone Megamek client
Windows 10 OS
Java version 11.0.12

### Description
While hosting a server for a game, an uncaught exception occurred during a match halfway through said match. The exception occurred at the end of a melee phase.

### Files
Player name Bows3r
[megamek.log](https://github.com/MegaMek/megamek/files/8767468/megamek.log)
[aaa.sav.gz](https://github.com/MegaMek/megamek/files/8767474/aaa.sav.gz)

**Comment by Windchild292** [comment]
```
19:29:59,727 ERROR [megamek.MegaMek] {AWT-EventQueue-0}
megamek.MegaMek.lambda$main$0(MegaMek.java:59) - Uncaught Exception Detected
java.lang.NullPointerException
	at megamek.common.actions.AbstractAttackAction.toDisplayableString(AbstractAttackAction.java:239)
	at megamek.client.ui.swing.AccessibilityWindow$1.gameNewAction(AccessibilityWindow.java:139)
	at megamek.common.event.GameNewActionEvent.fireEvent(GameNewActionEvent.java:52)
	at megamek.common.Game.processGameEvent(Game.java:3031)
	at megamek.common.Game.addAction(Game.java:2230)
	at megamek.client.Client.receiveAttack(Client.java:1189)
	at megamek.client.Client.handlePacket(Client.java:1546)
	at megamek.client.Client$1.lambda$packetReceived$0(Client.java:168)
```

**Comment by Windchild292** [comment]
Resolved in #3640
